### PR TITLE
Expose to occ the mothods for selective cart

### PR DESCRIPTION
**Pull request by fer9972**
the selective cart extension is required and the methods are exposed in the occ controller.

### CODE REVIEW

File: core-customize/tccommercewebservices/web/src/com/taloscommerce/webservices/v2/controller/SelectiveCartController.java [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+    @ApiBaseSiteIdParam
+    public void addToWishListFromCart(
+            @ApiFieldsParam @RequestParam(defaultValue = DEFAULT_FIELD_SET) final String fields,
+            @ApiParam(value = "entryNumber", required = true) @PathVariable final String entryNumber)
+            throws CommerceCartModificationException
+    {
+
+    }
+
+    @PostMapping(value = "/addToWishLis/{productCodes}", consumes = {MediaType.APPLICATION_JSON_VALUE, MediaType.APPLICATION_XML_VALUE})
```
Comment (JsonVM) [inline core-customize/tccommercewebservices/web/src/com/taloscommerce/webservices/v2/controller/SelectiveCartController.java]: U should rename that endpoint link and write: "/addToWishList/{productCodes}", u miss a 't'

File: core-customize/tccommercewebservices/web/src/com/taloscommerce/webservices/v2/controller/SelectiveCartController.java [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+            @ApiFieldsParam @RequestParam(defaultValue = DEFAULT_FIELD_SET) final String fields,
+            @ApiParam(value = "productCode", required = true) @PathVariable final String productCode)
+            throws CommerceCartModificationException
+    {
+
+    }
+
+    @GetMapping(value = "/getWishList", consumes = MediaType.APPLICATION_JSON_VALUE)
+    @ResponseStatus(value = HttpStatus.OK)
+    @ApiOperation(nickname = "getWishList", value = "get the wishlist")
```
Comment (JsonVM) [inline core-customize/tccommercewebservices/web/src/com/taloscommerce/webservices/v2/controller/SelectiveCartController.java]: Wich WishList?

**Review by JsonVM — CHANGES_REQUESTED** [review]

File: core-customize/tccommercewebservices/web/src/com/taloscommerce/webservices/v2/controller/SelectiveCartController.java (hunk 1 above)
Comment (Carlangasv) [inline core-customize/tccommercewebservices/web/src/com/taloscommerce/webservices/v2/controller/SelectiveCartController.java]: The mapping value has a typo

**Review by Carlangasv — CHANGES_REQUESTED** [review]

File: core-customize/tccommercewebservices/web/src/com/taloscommerce/webservices/v2/controller/SelectiveCartController.java (hunk 1 above)
Comment (fer9972) [reply core-customize/tccommercewebservices/web/src/com/taloscommerce/webservices/v2/controller/SelectiveCartController.java]: you right, I'll correct it

Comment (fer9972) [reply core-customize/tccommercewebservices/web/src/com/taloscommerce/webservices/v2/controller/SelectiveCartController.java]: you right, I'll correct it.

File: core-customize/tccommercewebservices/web/src/com/taloscommerce/webservices/v2/controller/SelectiveCartController.java (hunk 2 above)
Comment (fer9972) [reply core-customize/tccommercewebservices/web/src/com/taloscommerce/webservices/v2/controller/SelectiveCartController.java]: is the wishlist related to the current cart, so I'm going to change the value of the ApiOperation.

**Review by Carlangasv — APPROVED** [review]

**Review by JsonVM — APPROVED** [review]

**Pull request merged**